NBitcoin/BuilderExtensions/MiniscriptBuilderExtension.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NBitcoin.Miniscript;
+
+namespace NBitcoin.BuilderExtensions
+{
+	public class MiniscriptBuilderExtension : BuilderExtension
+	{
+		// Probably it is impossible to Combine items in generic way.
+		public override bool CanCombineScriptSig(Script scriptPubKey, Script a, Script b)
```

Comment:
This is the part I didn't know what to do.
Combining ScriptSig in Miniscript seems impossible (since there is no straightforward way to tell which [signature|Secret] corresponds to which [pubkey|Secret hash] in the script.) It seems we need some non-trivial refactoring on TransactionBuilder side.

Reply:
Combining ScriptSig is not too important. Especially we now have PSBT.